swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -9,12 +9,16 @@
 import Foundation
 import FullStory
 
-/**
- FullStory Manager is a file full of functions and enums that can make the FullStory API easier to apply to areas of your project.
+/*
+FullStory Manager provides common functions and artifacts that make it easier to integrate FullStory into your iOS app.
+    - Events
+    - LogLevel
+    - Identifying a user
     - create events
-    - logging and unmasking elements
+    - PrivacySetting: EXCLUDE, MASK, and UNMASK elements
  */
 
+/// The Event enum represents major events in an iOS app. These may differ based on your particular app.
 enum Event: String {
     case browsing
```

Comment:
An observation: it can be helpful to keep enum items in alphabetic order. This is more of a style things, so feel free to make the change or not.

Reply:
Thats makes sense.🙌🏾